obspy/io/hypodd/tests/test_pha.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from future.builtins import *  # NOQA
+
+import os
+import unittest
+
+from obspy import read_events, read_inventory
+from obspy.io.hypodd import pha
+
+
+class PHATestCase(unittest.TestCase):
+    """
+    Test suite for obspy.io.hypodd.pha
+    """
+
```

Comment:
Can you add a test to fully check all the values in the resulting catalog object?